test/perf/iterate.test.ts

```diff
@@ -27,3 +37,82 @@ describe("iterate", () => {
     }
   });
 });
+
+describe("readonly values - iterator vs array", () => {
+  setBenchOpts({
+    maxMs: 30 * 1000,
+    minMs: 10 * 1000,
+    runs: 1000,
+  });
+
+  const length = 250_000;
+  const balances = createBalanceList(length);
+
+  itBench("basicListValue.readonlyValues()", () => {
+    for (const balance of balances.readonlyValues()) {
+      balance + 1;
+    }
+  });
+  itBench("basicListValue.readonlyValuesArray()", () => {
+    const balancesArray = balances.readonlyValuesArray();
+    for (let i = 0; i < balancesArray.length; i++) {
+      balancesArray[i] + 1;
+    }
```

Comment:
This comparision is not totally right, since for the array version it iterates twice. A more proper would be
```ts
  itBench("basicListValue.readonlyValues()", () => {
    Array.from(balances.readonlyValues())
  });
  itBench("basicListValue.readonlyValuesArray()", () => {
    balances.readonlyValuesArray();
  })
```